### PR TITLE
Added digging holes chore

### PR DESCRIPTION
**Pull request by bakozetra**
Hey team,

Please have a look at my SD,add one file,happy to changes it if needs.

Thanks for advance
Synthia

### CODE REVIEW

**Comment by Fodilahy-mena** [comment]
Hi,it's cool but your verb to change is wrong.

**Comment by bakozetra** [comment]
Thanks for teaching .
synthia

File: # Sweeping-class.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (bakozetra) [inline # Sweeping-class.md]: Thanks for teaching Jerome

**Review by bakozetra — COMMENTED** [review]
Hey team,take a look at our PR,could you make any change if it needs,we would be so happy!!.


Your best,
Jerome & Synthia

**Review by voromahery — APPROVED** [review]
You did great!Well done!

**Pull request merged**